packages/slice-machine/components/AppLayout/Navigation/Menu/Navigation/ChangesIndicator.tsx

```diff
@@ -0,0 +1,30 @@
+import { Flex } from "theme-ui";
+
+interface ChangesIndicatorProps {
+  numberOfChanges: number;
+}
+
+export const ChangesIndicator = ({
+  numberOfChanges,
+}: ChangesIndicatorProps) => {
+  if (numberOfChanges === 0) {
+    return null;
+  }
+  return (
+    <Flex
+      sx={{
+        borderRadius: "50%",
+        backgroundColor: "#6548FF1A",
+        width: "24px",
+        height: "24px",
+        justifyContent: "center",
+        alignItems: "center",
+        fontSize: "12px",
+        color: "#5D40F7E5",
+        fontWeight: 600,
+      }}
+    >
+      {numberOfChanges}
```

Comment:
I think there was some logics about displaying "5+" if there are more than 5 changes.

Reply:
I think I remember some discussion about it during the sprint planning, Louis would know better.

Reply:
<img width="594" alt="Screenshot 2022-07-29 at 09 52 27" src="https://user-images.githubusercontent.com/69536010/181722517-c93babd0-a510-4ec1-bde9-570eb986a090.png">
Yup, i only implemented the design not the number logic here as the acceptance criteria says for this ticket says this

Reply:
ok, if you think that's enough. I hope one of the ticket describe that behavior or we might never do it 😅

Reply:
we have a ticket for this :) https://linear.app/prismic/issue/SM-731/aauser-on-the-navigation-i-can-see-the-number-of-changes-i-have